dev-lang/ghc/ghc-8.10.6.ebuild

```diff
@@ -74,7 +77,7 @@ BUMP_LIBRARIES=(
 
 LICENSE="BSD"
 SLOT="0/${PV}"
-#KEYWORDS="~amd64 ~x86"
+KEYWORDS="~amd64 ~x86"
```

Comment:
Should `~ppc64` be added as well?

Reply:
No, I was not able to get <9 building on ppc